.github/workflows/integration-tests.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
       opensearch:
-        image: registry.cern.ch/cern-sis/inspire/opensearch
```

Comment:
@DonHaul we have to make a separate PR for this change and merge it first.

Reply:
Separate PR here:
https://github.com/inspirehep/backoffice/pull/39/files